src/req/regional_requester.rs

```diff
@@ -110,7 +112,23 @@ impl RegionalRequester {
                             && !status.is_server_error())
                         {
                             log::debug!("Response {} (retried {} times), returning.", status, retries);
-                            break Err(RiotApiError::new(err, retries, Some(response), Some(status)));
+
+                            // Extract the response body from bytes into a String,
+                            // accounting for potentially non-utf-8 characters.
+                            let content = response.bytes().await;

```

Comment:
LGTM with one question, would `.text()` be better than `.bytes()` here?

Reply:
good point, one thing to keep in mind is that the underlying [`text()`](https://docs.rs/reqwest/0.10.4/src/reqwest/async_impl/response.rs.html#149) implementation is more complex. on the other hand it's probably more tested... 

honestly, I have no strong preference, so `text()` is a good choice then